downstream/assemblies/platform/assembly-controller-custom-credentials.adoc

```diff
@@ -0,0 +1,34 @@
+[id="assembly-controller-custom-credentials"]
+
+= Custom Credential Types
+
+As a system administrator, you can define a custom credential type in a standard format using a YAML/JSON-like definition,
+permitting the assignment of new credential types to jobs and inventory updates. 
+You can define a custom credential type that works in ways similar to existing credential types, such as a custom credential type that injects an API token for a third-party web service into an environment variable, for your playbook or custom inventory script to consume.
+
+Custom credentials support the following ways of injecting their authentication information:
+
+* Environment variables
+* Ansible extra variables
+* File-based templating, that is, generating `.ini` or `.conf` files that contain credential values
+
+You can attach one SSH and multiple cloud credentials to a Job Template.
+Each cloud credential must be of a different type. 
+Only one AWS credential or one GCE credential, etc., are permitted. 
```

Comment:
Avoid the use of "etc.". Use "and so on" or maybe just "...one GCE credential is permitted."